include/experimental/__p2630_bits/submdspan.hpp

```diff
@@ -0,0 +1,17 @@
+#include "submdspan_extents.hpp"
+#include "submdspan_mapping.hpp"
+
+namespace std {
+namespace experimental {
+template <class ElementType, class Extents, class LayoutPolicy,
+          class AccessorPolicy, class... SliceSpecifiers>
+constexpr auto
+submdspan(const mdspan<ElementType, Extents, LayoutPolicy, AccessorPolicy> &src,
+          SliceSpecifiers... slices) {
+  const auto sub_map_offset = submdspan_mapping(src.mapping(), slices...);
```

Comment:
Should we consider forwarding the slice specifiers, as in the code below?  It shouldn't make a performance difference for the current slice specifiers (that are all structs of small value types anyway); it's more a specification question.

```suggestion
template <class ElementType, class Extents, class LayoutPolicy,
          class AccessorPolicy, class... SliceSpecifiers>
constexpr auto
submdspan(const mdspan<ElementType, Extents, LayoutPolicy, AccessorPolicy> &src,
          SliceSpecifiers&&... slices) {
  const auto sub_map_offset = submdspan_mapping(src.mapping(), forward<SliceSpecifiers>(slices)...);
```

Reply:
we could try

Reply:
When I tried this I ran into some trouble with failing tests, i.e. some deduction went haywire, my guess is that all the internal logic also needs to account for && including all the helper functions for matching and counting etc.

Reply:
Thanks for trying!  I think it's OK to leave this alone for now.  If LWG wants it we can investigate it then.